lib/editor/actions/map/stopStrategies.js

```diff
@@ -284,7 +284,7 @@ export function addStopToPattern (pattern: Pattern, stop: GtfsStop | GtfsLocatio
         // Otherwise, check if a shape ought to be created. Then, save.
         if (patternHalts.length === 2 && followStreets && !!stop.stop_lon && !!stop.stop_lat &&
           // FLEX TODO: How to do this for locations?
-          stop.hasOwnProperty('stopId')
+          (stop.hasOwnProperty('stop_id') || stop.hasOwnProperty('stopId'))
```

Comment:
In my debugging I've seen both appear, but I think that's because some of the flex changes are still in flux. I'll add a comment stating that this is strange